README.md

```diff
@@ -155,7 +155,7 @@ SIG-specific GitHub discussions.
 | Name | Meeting Time | Meeting Notes | Slack Channel | Meeting Invites Group | [Governance Committee](./community-members.md#governance-committee) Liaison |
 |------|--------------|---------------|---------------|-----------------|--------------------------------|
 | Android: SDK + Automatic Instrumentation | Every Tuesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY) | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q) | [calendar-android](https://groups.google.com/a/opentelemetry.io/g/calendar-android) | [Trask Stalnaker](https://github.com/trask) |
-| Collector | Every Wednesday at 09:00 PT plus monthly on first Wednesday at 00:00 PT and on third Wednesday at 16:00 PT | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
+| Collector | Wednesdays, rotating between the following time slots: 00:00 UTC, 12:00 UTC, 16:00 UTC | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
```

Comment:
As if scheduling across time zones wouldn't already bring enough headaches, you'll also have to watch out for daylight savings time switches happening on different dates. The offset between regions will differ by +/-1h for 1-3 weeks each twice a year. To switch properly, this needs to be set up accordingly in the Google calendar entry by pinning it to UTC and not PT with US DST dates as we usually have it.

Reply:
Yes, I'll change the entries to use UTC instead.